modules/managing-certificates.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="managing-certificates_{context}"]
+= Managing certificates
+
+[role='_abstract']
+After installing {productname-short}, the {productname-long} Operator creates the `odh-trusted-ca-bundle` configuration file (ConfigMap) that contains the trusted CA bundle and adds it to all new and existing namespaces in the cluster. 
+By default, the {productname-long} Operator manages the `odh-trusted-ca-bundle` ConfigMap and automatically updates it if any changes are made to the CA bundle. You can choose to manage the `odh-trusted-ca-bundle` ConfigMap instead of allowing the (productname-long) Operator to manage it.
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+
+.Procedure
+. In the {openshift-platform} web console, click *Operators* → *Installed Operators* and then click the *{productname-long} Operator*.
+. Click the *DSC Initialization* tab.
+. Click the *default-dsci* object.
+. Click the *YAML* tab.
+. In the `spec` section, change the value of the `managementState` field for `trustedCABundle` to `Unmanaged`, as shown:
++
+[source]
+----
+spec:
+  trustedCABundle:
+    managementState: Unmanaged
+---- 
+. Click *Save*.
++
+Note that changing the `managementState` from `Managed` to `Unmanaged` does not remove the `odh-trusted-ca-bundle` ConfigMap, but the ConfigMap is not updated if you make changes to the `customCABundle` field.
+
+.Verification
+. In the `spec` section, set or change the value of the `customCABundle` field for `trustedCABundle`, for example:
```

Comment:
I am not clear on what is happening. Is it that the procedure steps are setting the management from auto-managed to self-managed? And that then, in the verification steps are showing that if you make a change (customCABundle: example123) in the DSC Initialization, the operator will not make that change.

I guess it would be good to also show how the user could "manually" make the change if the operator is not doing it. Would the user edit the `odh-trusted-ca-bundle` ConfigMap itself??